app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -101,15 +99,18 @@ class LeanplumMetricsService(
 
     @Suppress("ComplexMethod")
     override fun start() {
+        val preferencesLoaded = scope.async {
+                preferences = application.getSharedPreferences(PREFERENCE_NAME, MODE_PRIVATE)
+        }
 
         if (!application.settings().isMarketingTelemetryEnabled) return
```

Comment:
This will read from SharedPreferences under the hood: I'm surprised it doesn't set off StrictMode. Maybe we should move it into the launch statement too?